### PR TITLE
Updated lib/logitech_reciever/hidpp20.py with new keys

### PR DESCRIPTION
**Pull request by drupal-daffie**
Updated lib/logitech_reciever/hidpp20.py with new keys. Copied from the pull request from doctor64. See: https://github.com/pwr-Solaar/Solaar/pull/586.
I have compared it to the list from https://github.com/pwr-Solaar/Solaar/issues/582.

### CODE REVIEW

File: lib/logitech_receiver/hidpp20.py [hunk 1]
```diff
@@ -50,36 +51,63 @@
...
+	KEEP_ALIVE=0x0008,
 	RESET=0x0020, # "Config Change"
-	DFUCONTROL=0x00C0,
-	DFUCONTROL_2=0x00C1,
+	CRYPTO_ID=0x0021,
+	TARGET_SOFTWARE=0x0030,
+	WIRELESS_SIGNAL_STRENGTH=0x0080,
+	DFUCONTROL_LEGACY=0x00C0,
+	DFUCONTROL_UNSIGNED=0x00C1,
+	DFUCONTROL_SIGNED=0x00C2,
```
Comment (FFY00) [inline lib/logitech_receiver/hidpp20.py]: nitpick: Can you call them `DFU_CONTROL_*`?

Comment (drupal-daffie) [reply lib/logitech_receiver/hidpp20.py]: I gave them the names from the Logitech list. I am not sure what you want. Can you make the changes that you would like and then merge the pull request?

Comment (FFY00) [reply lib/logitech_receiver/hidpp20.py]: Sure. I was saying to rename `DFUCONTROL_LEGACY` to `DFU_CONTROL_LEGACY`, etc. But I will do that myself an merge.

Comment (drupal-daffie) [reply lib/logitech_receiver/hidpp20.py]: In the list from https://github.com/pwr-Solaar/Solaar/issues/582 they are like  `DFUCONTROL_LEGACY` and not like `DFU_CONTROL_LEGACY`.

Comment (FFY00) [reply lib/logitech_receiver/hidpp20.py]: I took that from #586, fixed now.

**Comment by pfps** [comment]
Is this request still current?

**Comment by robbat2** [comment]
@drupal-daffie please rebase

**Comment by FFY00** [comment]
This is no longer needed.

**Pull request closed without merging**